src/app/my-rewards/backers/components/RBI.tsx

```diff
@@ -100,5 +102,24 @@ export const RBI: FC<RBIProps> = props => {
   const {
     flags: { use_state_sync },
   } = useFeatureFlags()
-  return use_state_sync ? <RBIWStateSync {...props} /> : <RBIWTheGraph {...props} />
+
+  const {
+    data: stateSyncIsHealthy,
+    isLoading: healthCheckIsLoading,
+    error: healthCheckError,
+  } = useStateSyncHealthCheck({
+    initialData: true,
```

Comment:
I dont think that we need this property.

Reply:
yea fair

Reply:
yea fair. removed